src/Makefile.test.include

```diff
@@ -416,6 +416,31 @@ if EMBEDDED_UNIVALUE
 	$(AM_V_at)$(MAKE) $(AM_MAKEFLAGS) -C univalue check
 endif
 
+if ENABLE_BENCH
+if ENABLE_QT_TESTS
+check-valgrind: $(TEST_BINARY) $(BENCH_BINARY) test_bitcoin_qt
+else
+check-valgrind: $(TEST_BINARY) $(BENCH_BINARY)
+endif
+else
+if ENABLE_QT_TESTS
+check-valgrind: $(TEST_BINARY) test_bitcoin_qt
+else
+check-valgrind: $(TEST_BINARY)
+endif
+endif
+	@echo "Using the valgrind memory error detector: expect an ~50x slowdown, valgrind 3.14 or later required"
```

Comment:
- suggestion: s/Using the valgrind memory error detector/Running tests with valgrind memcheck/

- "~50x slowdown": looking through the manual and SE comments, I see a wide range of perf reports. The manual's [Running your program under Memcheck](http://valgrind.org/docs/manual/quick-start.html#quick-start.mcrun) section states "Your program will run much slower (eg. 20 to 30 times) than normal, and use a lot more memory." Replies to [this SE question](https://stackoverflow.com/questions/29365611/does-running-valgrind-slow-down-my-application) both mention a factor of 10 slowdown.

    Perhaps "expect a 10-50x slowdown and ~2x memory usage, vagrind 3.14 or later required." 

    This also avoids the awkward "a" or "an" question before the "~50x".

In summary, perhaps something like:

"Running tests with valgrind memcheck: expect a 10-50x slowdown and 2x memory usage, vagrind 3.14 or later required"

Reply:
That's better. Thanks! Now updated. Please re-review :)